pyproject.toml

```diff
@@ -138,6 +138,7 @@ runner = [
 ]
 # Janitor service (./Dockerfile_site)
 site = [
+    "aiohttp-debugtoolbar",
```

Comment:
This seems like an unnecessary dependency for production deployments.

Reply:
I do agree, but otherwise the container image doesn't fully function correctly without it (going on `--help` output) - not able to use all the flags/options shown